microsoft-edge/progressive-web-apps-chromium/how-to/app-actions.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Enable App Actions on Windows for PWA
+description: How to author and test an app action for the App Actions on Windows framework, for a Progressive Web App (PWA) that you've created.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: pwa
+ms.date: 05/19/2025
+---
+# Enable App Actions on Windows for PWA
+<!-- h2/h4/h5/h6 -->
+
+**Detailed contents:**
+
+* [Introduction](#introduction)
+   * [What's App Actions on Windows](#whats-app-actions-on-windows)
+   * [Why should you build an Action?](#why-should-you-build-an-action)
+* [Get started](#get-started)
+   * [Define the actions your PWA supports](#define-the-actions-your-pwa-supports)
+      * [Prepare ActionsManifest.json](#prepare-actionsmanifestjson)
+         * [Understanding the fields in ActionsManifest](#understanding-the-fields-in-actionsmanifest)
+         * [Supported Entity types for PWA](#supported-entity-types-for-pwa)
+      * [How does Windows App Actions framework call App Actions in a PWA?](#how-does-windows-app-actions-framework-call-app-actions-in-a-pwa)
+         * [PWA launch through URI Activation](#pwa-launch-through-uri-activation)
+         * [URI scheme](#uri-scheme)
+         * [Receiving ValueSet data from Windows and mapping it to Share Target](#receiving-valueset-data-from-windows-and-mapping-it-to-share-target)
+      * [Prerequisites for your PWA](#prerequisites-for-your-pwa)
+         * [Web Manifest validation for protocol_handlers](#web-manifest-validation-for-protocol_handlers)
+         * [Web Manifest validation for share_target](#web-manifest-validation-for-share_target)
+         * [Web Manifest validation for launch_handler](#web-manifest-validation-for-launch_handler)
+         * [Accept entity info from Windows App Actions framework](#accept-entity-info-from-windows-app-actions-framework)
+   * [Package your PWA](#package-your-pwa)
+      * [How to package store PWA in Edge Stable](#how-to-package-store-pwa-in-edge-stable)
+      * [How to package store PWA in Edge Canary](#how-to-package-store-pwa-in-edge-canary)
+   * [Testing your PWA in Windows](#testing-your-pwa-in-windows)
+      * [Using Windows actions test tool](#using-windows-actions-test-tool)
+
+
+<!-- ====================================================================== -->
+## Introduction
+
+Here's how to author and test an app action for the App Actions on Windows framework, for a Progressive Web App (PWA) that you've created.
+
+
+<!-- ------------------------------ -->
+#### What's App Actions on Windows
+
+An App Action on Windows is an individual unit of behavior that a Windows app can implement and register.  The App Action can then be accessed from other apps and experiences, seamlessly integrating into user workflows.
```

Comment:
I think we should add a link somewhere in this section like "To learn more about App Actions on Windows, see [link]".